src/app/templates/map_controls/category-dropdown.html

```diff
@@ -0,0 +1 @@
+<div class="toolbar-control-button"><i class="<%= content %>"></i></div>
```

Comment:
is there any other case of a 1-line template? There's so little to this I'd say we just inline as a string with the js, but only if there's another example within this codebase of it being inlined.

(General principle: fly in formation with whatever's already going on. If this is normal for the codebase, so be it).

Reply:
There seem to be several single line templates in the codebase. Is this bad practice?

Reply:
that's fine then. I'm generally of the opinion that fewer files is better but if they're doing it already it's fine.